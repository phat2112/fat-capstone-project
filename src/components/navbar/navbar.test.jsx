import { render } from "@testing-library/react";
import { describe, it, expect } from "vitest";
import Navbar from "./";

describe("Navbar", () => {
  it("renders Navbar", () => {
    const { asFragment } = render(<Navbar />);
    const rendered = asFragment();

    console.log(rendered);
    expect(rendered).toMatchSnapshot();
    // check if App components renders headline
  });
});
