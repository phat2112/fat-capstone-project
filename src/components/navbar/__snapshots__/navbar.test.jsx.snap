// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Navbar > renders Navbar 1`] = `
<DocumentFragment>
  <div
    class="css-1q16c1q"
  >
    <img
      alt="logo"
      class="chakra-image css-0"
      src="/src/assets/Logo.png"
    />
    <div
      class="css-1xmwlc4"
    >
      <p
        class="chakra-text css-0"
      >
        Home
      </p>
      <p
        class="chakra-text css-0"
      >
        Order
      </p>
      <p
        class="chakra-text css-0"
      >
        Cart
      </p>
    </div>
  </div>
</DocumentFragment>
`;
